sonar-kotlin-plugin/src/main/resources/static/documentation.md

```diff
@@ -12,6 +12,22 @@ key: kotlin
 
 You can discover and update Kotlin-specific [properties](/analysis/analysis-parameters/) in:  <!-- sonarcloud -->Project <!-- /sonarcloud -->**[Administration > General Settings > Kotlin](/#sonarqube-admin#/admin/settings?category=kotlin)**.
 
+## Kotlin Analysis and Bytecode
+If you are not using the Maven or Gradle Sonar plugins, it is strongly recommended to provide the paths of all dependency binaries used by
```

Comment:
If you are not using the `SonarScanner for Maven` or `SonarScanner for Gradle`, it is strongly ...